### DISCUSSION TITLE
About Config

### DISCUSSION THREAD

**Discussion by mgesmundo** [nuvolaris/nuvolaris#91] [Config]
Hi there,
your project is awesome! Yes I know: this is the same starting comment in every open source project. But, really: I think this is a very promising project!
Well. I tried to found a document describing the goals of this frontend/backend without success.
I'm evaluating to contribute to this project (if you like) but I can't find a use cases list, a features list, etc.
For every frontend project I create a Sketch mockup to evaluate the UX/UI before start coding.
Do you have any useful information for this purpose?
Thank you.
All the best

**Comment by msciabarra** [comment]
THere is not a frontend but a configurator, an application that will allow to "administer" OpenWhisk more easily instead of relying on the command line wskadmin. You can find what we want to do [filtering the issues for the subproject `nuvolaris-config`](https://github.com/nuvolaris/nuvolaris/issues?q=is%3Aopen+is%3Aissue+label%3Anuvolaris-config)
